### ISSUE TITLE
Wrong parameters for Fadion\Fixerio\Exceptions\ResponseException([string $message [, long $code [, Throwable $previous = NULL]]])

### ISSUE THREAD

**Issue by spagu** [fadion/Fixerio#15]
I did upgrade to version to 2 also created a key but now I am facing this problem.

How to solve?

this is happening on first example from your manual.

**Comment by Nacoma** [comment]
This was fixed with #12.

You may need to update your dependency version of this package to `dev-master` until a release is made.

**Comment by spagu** [comment]
updated. Now the problem is : 
```
Type: Fadion\Fixerio\Exceptions\ResponseException
Code: 105
File: /project/vendor/fadion/fixerio/src/Exchange.php
Line: 276

```

**Comment by Nacoma** [comment]
According to their API docs a `105` is `Your current subscription doesn’t support this endpoint`. The response exception is supposed to be thrown in this case.

Is the corresponding error message not there?

**Comment by spagu** [comment]
Yeah I think so. I was doing USD to GBP and EUR. So no more free service here. That's a problem for as we would have to develop own lib for that.

**Comment by Nacoma** [comment]
It's simple enough to convert to another base. You could do so using [this well designed PHP money library](https://github.com/moneyphp/money) or simply re-base the rates yourself using  something like so:

```php
$rates = [
     'USD' => .9,
     'EUR' => 1,
    // ...
];

$base = 'USD';

$rebasedConversionRates = array_map(function (string $rate) use ($base) {
    return bcdiv($rate, (string) $base, 16);
}, $rates);
```

I'm currently working on an integration library for fixers API and the money lib but that's a few weeks out still.
